app_dart/lib/src/request_handlers/scheduler/scheduler_request_subscription.dart

```diff
@@ -108,16 +162,19 @@ class SchedulerRequestSubscription extends SubscriptionHandler {
 
     for (bbv2.BatchResponse_Response batchResponseResponse in response.responses) {
       if (batchResponseResponse.hasScheduleBuild()) {
-        retry.removeWhere((element) => batchResponseResponse.scheduleBuild.builder == element.scheduleBuild.builder);
+        retry.remove(
+          retry.firstWhere((element) => batchResponseResponse.scheduleBuild.builder == element.scheduleBuild.builder),
+        );
       } else {
         log.warning('Response does not have schedule build: $batchResponseResponse');
+        errors.add('$batchResponseResponse');
```

Comment:
Are the toString()'s of these objects readable?

Reply:
Define readable... They are strings. The errors I've seen are 'builder does not exist' which is certainly helpful.

Reply:
Ah ok, I just meant they aren't like `Instance of Foo`. Sg.